### ISSUE TITLE
Onset/Frames Question:evaluation metric results different to those in  the paper

### ISSUE THREAD

**Issue by nkjulia** [magenta/magenta#1928]
Hi, i'm evaluating pretrained onset and frames transcription model using maestro-v3.0.0 test dataset and the pretrained model (https://github.com/magenta/magenta/tree/master/magenta/models/onsets_frames_transcription#pre-trained)
but i found my evaluation result was not  as good as those in the paper.

below is my evaluation result: 

Saving dict for global step 1000: global_step = 1000, loss = 0.16057782, losses/frame = 0.072237805, losses/offset = 0.020498512, losses/onset = 0.011072341, losses/velocity = 0.05706539, metrics/frame_accuracy = 0.9819877, metrics/frame_accuracy_without_true_negatives = 0.71850765, metrics/frame_f1_score = 0.83273304, metrics/frame_precision = 0.8868856, metrics/frame_recall = 0.7936476, metrics/note_density = 8.9426155, metrics/note_f1_score = 0.89242053, metrics/note_precision = 0.9401742, metrics/note_recall = 0.855884, metrics/note_with_offsets_f1_score = 0.664754, metrics/note_with_offsets_precision = 0.6990717, metrics/note_with_offsets_recall = 0.6382193, metrics/note_with_offsets_velocity_f1_score = 0.385653, metrics/note_with_offsets_velocity_precision = 0.40885562, metrics/note_with_offsets_velocity_recall = 0.36769807, metrics/note_with_velocity_f1_score = 0.5094813, metrics/note_with_velocity_precision = 0.54153734, metrics/note_with_velocity_recall = 0.48474383

can anyone tell me why? thx.

**Comment by cghawthorne** [comment]
I'm not sure exactly what all is happening here, but keep in mind that the original MAESTRO paper had results for MAESTRO v1, which is not comparable to MAESTRO v3 because they have different test sets.

**Comment by nkjulia** [comment]
> TRO paper had results for MAESTRO v1, which is not comparable to MAESTRO v3 because they have different test sets.

thank u :) 
I'll  reevaluate on the test data of V1

**Comment by nkjulia** [comment]
> I'm not sure exactly what all is happening here, but keep in mind that the original MAESTRO paper had results for MAESTRO v1, which is not comparable to MAESTRO v3 because they have different test sets.

i did reevaluate the test data of v1 , but metrics still quite different with those in the  Paper which are much better....
 any advance?  thx.
